src/main/java/org/testeditor/fixture/web/json/BrowserSetupElement.java

```diff
@@ -0,0 +1,115 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2017 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+
+package org.testeditor.fixture.web.json;
+
+import java.util.List;
+
+import com.google.gson.JsonObject;
+
+public class BrowserSetupElement {
+
+    private String browserSetupElementName;
+    private String osName ;
+    private String osVersion;
+    private String browserName;
+    private String browserVersion;
+    private JsonObject settingsAsJsonObject; 
```

Comment:
what exactly do I need this for?

Reply:
this is for the extraction of all settings and to separate these into capabilities, options, osVersion, browserName, etc.